README.md

```diff
@@ -211,34 +221,69 @@ Link to other services:
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
+# How to get an output
+
+There are 4 possible outcomes depending on the input:
+
+1. The input is a valid payment file.
+Outcome:
+    - The transactions are logged,
+    - The payment file is moved from `inbound` to `archive`,
+    - Each valid payment request is sent to the Topic `ffc-pay-request`.
+
+2. The input is a payment file where the amount of payment requests does not match the batch header.
+Outcome: 
+    - An error message is logged,
+    - The payment file is moved from `inbound` to `quarantine`.
+
+3. The input is a payment file with an invalid batch header (e.g. wrong date format)
+Outcome: 
+    - An error message is logged,
+    - The payment file is moved from `inbound` to `quarantine`.
+
+4. The input is a payment file where the batch header value matches the payment line values but one or more payment line does not match the corresponding invoice line total.
+Outcome:
+    - The valid payment requests are logged and error messages are logged for each invalid request,
+    - The payment file is moved from `inbound` to `archive`,
+    - Each valid payment request is sent to Topic `ffc-pay-request`,
+    - An invalid event is sent to `ffc-pay-event` for each invalid payment request.
+
```

Comment:
There is another outcome where the batch sequence does not match the expected.  If the sequence is higher than expected it is ignored, if it is lower then it is quarantined.

Perhaps we should have a section on batch sequences?